src/network/tarcap/packets_handlers/new_dag_block_packet_handler.cpp

```diff
@@ -42,11 +40,15 @@ void DagPacketsHandler::process(const PacketData &packet_data, const std::shared
   }
 
   // Ignore new block packets when pbft syncing
-  if (syncing_state_->is_pbft_syncing()) return;
+  if (syncing_state_->is_pbft_syncing()) {
+    LOG(log_dg_) << "Ignore new dag block " << hash.abridged() << ", pbft syncing is on";
+    return;
+  }
```

Comment:
move this to line 34

Reply:
why ? you might be syncing 1 latests pbft block you can mark those txs and block as known ?

Reply:
+ these txs are just extra txs that the sending peer thinks you dont have so not that much parsing anyway, I would keep it this way